workitems/tokenlist/l2-token-list-v1.0-psd01.md

```diff
@@ -601,9 +621,12 @@ This document defines the conformance levels of a canonical token list as follow
 #### **[D3]** 
 A claim that a canonical token list implementation conforms to this specification SHOULD describe the testing procedure used to justify the claim.
 
+[D3]](#d3) testability: Since each of the non-conformance-target requirements in this documents is testable, so must be the totality of the requirements in this document.
```

Comment:
This is testable based on whether or not there is a description.

The apparent requirement is actually that a description of the procedure used provides a justification for claiming conformance to each requirement applicable, and the testability requires a person familiar with the specification to determine that the test matches the requirement.

Reply:
addressed, see new commit @chaals